libs/accelerator/package.json

```
@@ -5,7 +5,8 @@
   "peerDependencies": {
     "tslib": "^2.6.3",
     "rxjs": "^7.8.1",
-    "@nx/devkit": "^20.3.0"
+    "@nx/devkit": "^20.3.0",
+    "@onecx/nx-migration-utils": "^6.0.0-rc.8"
```

Why don't you use the same version as for other libs here?
```suggestion
    "@onecx/nx-migration-utils": "^6.0.0-rc.10"
```